src/sst/core/params.cc

```diff
@@ -497,7 +497,7 @@ Params::getGlobalParamSet(const std::string& name)
 std::vector<std::string>
 Params::getGlobalParamSetNames()
 {
-    std::vector<std::string> ret;
+    std::vector<std::string> ret(global_params.size());
```

Comment:
I meant `std::vector<T>::reserve` https://en.cppreference.com/w/cpp/container/vector/reserve, 

This code will construct a vector of size `global_params.size()` which will have default constructed strings and then additionally push back another `size` number of strings.

Reply:
Arg.  Thanks.  I mess that up from time to time with vectors.  I'll go back and fix it.